components/engine_rocks/src/status.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+pub fn from_rocksdb_error(msg: impl Into<String>) -> engine_traits::Error {
+    // TODO: use correct code.
+    engine_traits::Error::Engine(engine_traits::Status::with_error(
+        engine_traits::Code::IoError,
+        msg,
+    ))
+}
+
+pub fn from_engine_traits_error(s: engine_traits::Error) -> String {
+    format!("{:?}", s)
+}
+
+/// A macro that will transform a rocksdb error to engine trait error.
+///
+/// r stands for rocksdb, e stands for engine_trait.
+macro_rules! r2e {
```

Comment:
what about rocks2engine_error so it's no need to see the definition to understand what it does.

Reply:
The name is too long to be written everywhere.